content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,45 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The Control Plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside a group of containers called
+a {{< glossary_tooltip text="_pod_" term_id="pod" >}}. The idea of a pod
+is to isolate different workloads from each other. The computers
+that run pods are called _nodes_, and one node can run pods from many
+different workloads.
```

Comment:
Either "workloads" should be changed to "applications", or the use of "application" above should be changed to "workload". Since "workload" is used throughout, I'd lean towards using it everywhere.

Reply:
The way I see the terms used, “workload” and “application” are not synonyms. An example:

> This workload has a JavaScript SPA client-side component that interacts via HTTP/3 with two components, FOO and BAR. FOO consists of three stateless applications that use the BAR event bus to share and store application state.
> You can deploy the stack to Kubernetes using the EXAMPLEFOOBAR operator, or run it locally with a container runtime such as Docker.

if you swap “workload” and “application” it doesn't read right. An application could make up a whole workload, or it could be just one part.